pychef/__init__.py

```diff
@@ -33,6 +32,25 @@
     dose_step = None
       .type = float(value_min=0)
   }
+  experiments
```

Comment:
These parameters are only applicable to the command line program, and if experiments/reflections are given as input, not for mtz files. Perhaps these parameters should live in the command line phil scope? As suggested below these parameters could be passed explicitly to the `interpret_images_to_doses_options` function.